src/test/scala/cromwell/binding/expression/TypeEvaluatorSpec.scala

```diff
@@ -18,8 +18,8 @@ class TypeEvaluatorSpec extends FlatSpec with Matchers {
 
   def identifierLookup(String: String): WdlType = {
     String match {
-      case "cgrep" => WdlCallOutputsObjectType(namespace.workflow.calls.find(_.name == "cgrep").get)
-      case "ps" => WdlCallOutputsObjectType(namespace.workflow.calls.find(_.name == "ps").get)
+      case "cgrep" => cromwell.binding.types.WdlCallOutputsObjectType(namespace.workflow.calls.find(_.name == "cgrep").get)
+      case "ps" => cromwell.binding.types.WdlCallOutputsObjectType(namespace.workflow.calls.find(_.name == "ps").get)
```

Comment:
I got a bunch of "duplicate import" errors before. They seem to have addressed themselves now so I'll remove these